### PR TITLE
Website: Update to link to Jetpack installation

### PR DESCRIPTION
**Pull request by faxu**

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -517,7 +517,7 @@ function commandMessage(key) {
             "Follow build instructions from&nbsp;<a href='https://aka.ms/build-ort-tensorrt' target='_blank'>here</a>",
 
         "linux,Python(3.5-3.7),ARM64,TensorRT":
-            "Follow build instructions from&nbsp;<a href='https://aka.ms/build-ort-tensorrt' target='_blank'>here</a>",
+            "For Jetpack, follow installation instructions from <a href='https://elinux.org/Jetson_Zoo#ONNX_Runtime' target='_blank'>here</a>",
```
Comment (faxu) [inline js/script.js:520]: @manashgoswami do you suggest to use "Jetpack" or "Jetson" to refer to this compatibility?

Comment (manashgoswami) [reply js/script.js:520]: Jetpack 4.4 or higher.
This package is for CUDA/cuDNN not for TensorRT.

File: js/script.js [hunk 2]
```diff
@@ -242,7 +242,10 @@ function commandMessage(key) {
 
         "windows,Python(3.5-3.7),X64,CUDA":
             "pip install onnxruntime-gpu",
-
+	
+	"linux,Python(3.5-3.7),X64,CUDA":
```
Comment (manashgoswami) [inline js/script.js:246]: change X64 to ARM64

Comment (faxu) [reply js/script.js]: sorry, i think i need my afternoon coffee today. 3rd time's the charm. :P

**Pull request merged**